ardere/aws.py

```diff
@@ -107,6 +114,10 @@ def s3_ready_file(self):
             key="{}.ready".format(self._plan_uuid)
         )
 
+    @property
+    def influx_db_name(self):
+        return "run-{}".format(self.plan_uuid)
```

Comment:
Should standardize the db name: it's "loads" + self.run.uuid.replace('-', '') in loads-broker, I guess influxdb is ok with dashes then? (loads-broker was already removing them before I got here)